10up/ElasticPress
# add hierarchical taxonomy support to root level tax_query

### Description of the Change

Changes in #1617 breaking WooCommerce category archives. It can potentially break any hierarchical taxonomy archives (IMO). Appending child slugs seemed the solution. 

@tlovett1 opinions on this?

### Verification Process

* Visit a WooCommerce parent product category.
* Make sure parent category has a child category and there are products belongs to child category only
* See the products on the main category

### Checklist:






### Applicable Issues

#1576 could be related
